docs/operator-manual/Rancher-Fleet.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: "Rancher Fleet"
+title: ""
+---
+
+# Managing Kubewarden with Rancher Fleet
+
+The Kubewarden Helm charts, as other Helm charts, can be managed via [Rancher
+Fleet](https://fleet.rancher.io/). Rancher Fleet uses Kubernetes CRDs to define
+a GitOps approach for managing Kubernetes clusters. It does this by defining
+Fleet Bundles.
+
+### Installing
+
+The Kubewarden charts are normal charts, and they have defined dependencies that
+need to be ready before the dependent chart is deployed:
+- First, install `cert-manager`, as it is needed for the Kubewarden stack default config.
+- Second, install `kubewarden-crds`, as it is needed for the rest of Kubewarden charts.
+- Third, install `kubewarden-controller`, as it is needed by `kubewarden-defaults`.
+- Fourth, install `kubewarden-defaults`.
+
+On Rancher Fleet, one can codify the chart dependencies using
+[`dependsOn` in the _fleet.yaml_ file](https://fleet.rancher.io/gitrepo-structure#fleetyaml).
+
+At the time of writing and by how Rancher Fleet works, one may see transient
+errors until the charts are ready, such as:
+```
+ErrApplied(1) [Cluster fleet-local/local: dependent bundle(s) are not ready:
+[kubewarden-example-helm-kubewarden-controller]]
+```
+These errors don't signify a problem, and once each chart has
+finished deploying, they will be gone.
+
+### Removing
+
+Uninstalling CRDs automatically is normally not supported in any tooling, and
+Rancher Fleet is no exception.
+
+If you want to perform a correct removal, make sure to remove first the Bundle
+for `kubewarden-defaults` from the cluster, then `kubewarden-controller`, and
+last, `kubewarden-crds`.
```

Comment:
Can we provide some copy and paste docs about that?

Reply:
It's not so easy. I can explain it by deleting the [`defaullts` bundle folder](https://github.com/viccuad/kubewarden-fleet/tree/main/helm-kubewarden/defaults) from the repo, committing it, waiting for Fleet to apply, removing the `controller` bundle folder, comitting, waiting for Fleet to apply, and removing the `crds` bundle folder, committing, and waiting for Fleet to apply.

Reply:
A better approach is to have the CRDs be in their own GitRepo, too. Done in https://github.com/kubewarden/docs/pull/161/commits/eae9a1cf5bf6643303c549ca2199971d47ae197c.